gripper/core/tasks.cpp

```diff
@@ -41,8 +41,13 @@ static auto i2c2_poll_client =
 static auto i2c3_poll_client =
     i2c::poller::Poller<freertos_message_queue::FreeRTOSMessageQueue>{};
 
-static auto capacitive_sensor_task_builder =
-    freertos_task::TaskStarter<512, sensors::tasks::CapacitiveSensorTask>{};
+static auto capacitive_sensor_task_builder_s0 =
```

Comment:
can we give these names that relate to the physical use? like `capacitive_sensor_rear` and `_front` or whatever